app/controllers/api/drafts_controller.rb

```diff
@@ -1,11 +1,13 @@
 class Api::DraftsController < Api::BaseController
 
   def create
-    draft = Email.new draft_params
+    draft = Email.new draft_params.merge(
```

Comment:
Can you experiment with making a class method on the `Email` model called `draft`?
Something like `def self.draft(params)` in the `Email` model hopefully would do the trick. Then, all it does is instantiate an email with `new` and then set it to be a draft.